### PR TITLE
Improving test section in README.md

### PR DESCRIPTION
**Pull request by nicolas2bert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,10 +89,27 @@ You can run the linter with:
 npm run lint
 ```
 
-You can run local functional tests with:
+You can run local functional tests:
+
+We use s3cmd and Redis for our functional tests.
```
Comment (bennettbuchanan) [inline README.md]: Maybe 'Redis' should be capitalized throughout?

Comment (nicolas2bert) [inline README.md]: Agree! Thank you.

File: README.md [hunk 2]
```diff
@@ -89,10 +89,29 @@ You can run the linter with:
 npm run lint
 ```
 
-You can run local functional tests with:
+You can run local functional tests:
```
Comment (rahulreddy) [inline README.md]: I would reword this as "Running functional tests locally"

File: README.md [hunk 3]
```diff
@@ -89,10 +89,29 @@ You can run the linter with:
...
+
+We use s3cmd and Redis for our functional tests.
```
Comment (rahulreddy) [inline README.md]: The test suite requires additional tools, **s3cmd** and **Redis** installed in the environment the tests are running in.

File: README.md [hunk 4]
```diff
@@ -89,10 +89,29 @@ You can run the linter with:
...
+
+* Install [s3cmd](http://s3tools.org/download)
+* Install [redis](https://redis.io/) and start Redis.
```
Comment (rahulreddy) [inline README.md]: It would be appropriate to link the users to the download url https://redis.io/download

File: README.md [hunk 5]
```diff
@@ -89,10 +89,29 @@ You can run the linter with:
...
+* Add localCache section to your `config.json`:
+
+```
+"localCache": {
+    "host": REDIS_HOST,
+    "port": REDIS_PORT
+}
+```
+
+where `REDIS_HOST` is your Redis instance ip address (`"127.0.0.1"` if your
```
Comment (rahulreddy) [inline README.md]: Redis instance ~~ip~~ IP address

File: README.md (hunk 2 above)
Comment (nicolas2bert) [inline README.md]: Thanks for the review - I addressed your comments.
